lib/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner.rb

```diff
@@ -4,14 +4,14 @@ module Script
   module Layers
     module Infrastructure
       module Languages
-        class AssemblyScriptTaskRunner
+        class AssemblyScriptTaskRunner < TaskRunner
           BYTECODE_FILE = "build/script.wasm"
           METADATA_FILE = "build/metadata.json"
           SCRIPT_SDK_BUILD = "npm run build"
 
           attr_reader :ctx, :script_name
 
-          def initialize(ctx, script_name)
+          def initialize(ctx, script_name) # rubocop:disable Lint/MissingSuper
             @ctx = ctx
             @script_name = script_name
```

Comment:
Couple questions here:

1. This is the same in every task runner. What do you think of moving this (and the `attr_reader`s) to the initializer instead? 
2. I don't think anything even uses `script_name` on the task runner anymore. If you do my point 1, maybe we can just 🔥 it as the same time

Reply:
Yup this works! I've moved the common initialize up and this also lets me drop the rubocop disable 😄 
Script_name was also fairly easy to remove... I wonder how long we've not needed it for? 🤔